lib/timeutils/wallclocktime.h

```diff
@@ -124,6 +124,33 @@ struct _WallClockTime
 
 #endif
 
+static inline
+void dump_wall_clock_time(const WallClockTime *wct, GString *output)
```

Comment:
again, this can go to the implementation file and please move "void" to the previous line.